Components/IModel.cs

```diff
@@ -216,6 +241,12 @@ public interface IModel
         /// </summary>
         FooterLink TermsConditionsLink { get; set; }
 
+        /// <summary>
+        /// Configures the Terms and Conditions Link in the footer
+        /// Set by application programmatically
+        /// </summary>
+        SubFooterLink TermsFooterLink { get; set; }
```

Comment:
Isn't that the same as TermsConditionsLink?

Reply:
Removed